manifests/init.pp

```diff
@@ -42,9 +48,31 @@
   $ca_file          = $::foreman_scap_client::params::ca_file,
   $host_certificate = $::foreman_scap_client::params::host_certificate,
   $host_private_key = $::foreman_scap_client::params::host_private_key,
+  $foreman_repo_rel = undef,
+  $foreman_rpm_key  = undef,
   $policies,
 ) inherits foreman_scap_client::params {
 
+  if $foreman_repo_rel {
+    $gpgkey_file = '/etc/pki/rpm-gpg/RPM-GPG-KEY-foreman'
+
+    file { $gpgkey_file:
+      ensure => present,
+      source => $foreman_rpm_key,
+      mode   => '0644',
+    }
+
+    yumrepo { 'foreman-plugins':
+      ensure   => present,
+      descr    => "Foreman plugins ${foreman_repo_rel}",
+      baseurl  => "http://yum.theforeman.org/plugins/${foreman_repo_rel}/el${::operatingsystemmajrelease}/\$basearch",
```

Comment:
NP, I see two possibilities, tell me witch you like best. 
1. `forman_plugins_repo_url = undef` param. Use it when set, otherwise set it to `yum.theforeman.org/plugins/`, build URL
2. we could use a hash for all three operating systems and resolve it by `osfamily`

If it doesn't matter, I would opt for (1)